SU-SOE/stanford_soe_helper
# SOE-1929: Fixing related spotlights to exclude current node

# NOT READY FOR REVIEW


# Summary
See https://stanfordits.atlassian.net/browse/SOE-3211
- This changes the authoring experience for the top banner (formerly the featured image) on the Stanford Page.
- This add the CTA button to support the "Make a Gift" button. See:
https://stanfordits.atlassian.net/browse/SOE-3158

# Needed By (Date)
- Sprint end

# Criticality
- Client is eagerly awaiting this for Giving pages


# Steps to Test
- Switch to this branch: `7.x-2.x-soe-1929-spotlight-related-exclude`
- Clear cache so Drupal can find the module: `drush cc all`
- Revert feature: `drush fr stanford_people_spotlight_views`
- Go to a spotlight person at: `jse-soe-dev/spotlight/ryan-cohen`
- Make sure the current node is not showing in the related spotlight block at the bottom of the page.

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)
QA: https://stanfordits.atlassian.net/browse/SOE-1929

## Related PRs

## More Information

## Folks to notify
@boznik 


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
